src/components/approvallist/ApprovalEntry.tsx

```diff
@@ -1,71 +1,79 @@
 
   return (
-    <Accordion key={approval.id}>
-      <AccordionSummary>
-        <ColumnGrid>
-          <FlexRowWrapper>
-            {/* small wrapper which stops the accordion from reacting to a checkbox click */}
-            <div onClick={(event) => event.stopPropagation()} style={{ width: 0 }}>
-              <Checkbox
-                checked={approval.selected}
-                label=""
-                name="markedForChange"
-                onChange={(event) => {
-                  event?.stopPropagation();
-                  approval.toggleSelected();
-                }}
-              />
-            </div>
-          </FlexRowWrapper>
-          <FlexRowWrapper>
-            <img
-              src={tokenMap?.get(approval.tokenAddress)?.logoUri}
-              width={24}
-              height={24}
-              alt={tokenMap?.get(approval.tokenAddress)?.symbol}
-            />
-            <div style={{ display: 'flex', flexDirection: 'column' }}>
-              <Text size="lg" strong>
-                {tokenMap?.get(approval.tokenAddress)?.symbol}
-              </Text>
-              <EthHashInfo textSize="sm" hash={approval.tokenAddress} shortenHash={4} showCopyBtn />
-            </div>
-          </FlexRowWrapper>
-          <FlexRowWrapper>
-            <EthHashInfo hash={approval.spender} shortenHash={4} showAvatar showCopyBtn />
-          </FlexRowWrapper>
-          <FlexRowWrapper>
-            {UNLIMITED_ALLOWANCE.isEqualTo(approval.currentAmountInWEI) ? (
-              <Text size="lg" strong>
-                Unlimited
-              </Text>
-            ) : (
-              <Text size="lg" strong>
-                {approval.currentAmount.toFixed()}
-              </Text>
-            )}
-          </FlexRowWrapper>
-        </ColumnGrid>
-      </AccordionSummary>
-      <ApprovalTransactionHistory tokenAddress={approval.tokenAddress} transactions={approval.transactions} />
-    </Accordion>
+    <ColumnGrid style={{ borderBottom: `1px solid ${theme.palette.border.main}` }}>
+      <FlexRowWrapper>
+        {/* small wrapper which stops the accordion from reacting to a checkbox click */}
+        <div onClick={(event) => event.stopPropagation()} style={{ width: 0 }}>
+          <Checkbox
+            checked={approval.selected}
+            name="markedForChange"
+            onChange={(event) => {
+              event?.stopPropagation();
+              approval.toggleSelected();
+            }}
+          />
+        </div>
+      </FlexRowWrapper>
+      <FlexRowWrapper>
+        <Avatar
+          src={tokenMap?.get(approval.tokenAddress)?.logoUri}
+          sx={{ width: '24px', height: '24px' }}
+          alt={tokenMap?.get(approval.tokenAddress)?.symbol}
+        />
+        <div style={{ display: 'flex', flexDirection: 'column' }}>
+          <Typography fontWeight={700}>{tokenMap?.get(approval.tokenAddress)?.symbol}</Typography>
```

Comment:
Nit: if you move the `symbol` getter outside of the render, you'd need not call this twice.